gdx/src/com/badlogic/gdx/utils/Queue.java

```diff
@@ -142,8 +141,7 @@ public T removeFirst () {
 
 		final T result = values[head];
 		values[head] = null;
-		head++;
```

Comment:
The main comment is on the if(++head ..) format which is an inc/dec operator inside the if loop.